### PR TITLE
Remove bash verbose traces

### PR DESCRIPTION
**Pull request by enriquebelarte**

### CODE REVIEW

**Comment by enriquebelarte** [comment]
Fixes https://issues.redhat.com/browse/MGMT-14438

File: tools/sro2kmm/sro2kmm [hunk 1]
```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env bash
 #
-set -ex
```
Comment (qbarrand) [inline tools/sro2kmm/sro2kmm:3]: Don't we want to keep `-e`? How about `-u`?

Comment (enriquebelarte) [reply tools/sro2kmm/sro2kmm:3]: Done.

**Pull request merged**